### PR TITLE
Fixed #656

### LINKED ISSUE #656: Readme.rst to be improvised
## Typo mistakes:
"Eclipes" - expected=>"Eclipse"
"Doumentation" - expected=>"Documentation"

## Improper punctuation:
"Bluemix ?"-(space not required)- expected=>"Bluemix?"
"Ocean ?"-(space not required)- expected=>"Ocean?"
"rss" - expected=>"RSS"

## Grammatical error:
"consist"- expected=>"consists"

I would like to fix them.

### PR DESCRIPTION
**Pull request by girijamanojkumarreddy**
Fixes #656 
Changes: corrected all the mistakes mentioned in #656

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -177,7 +177,7 @@ Where can I get the latest news about Susi AI?
 -------------------------------------------
 
 Hey, this is the tool for that! Just put
-http://loklak.org/api/search.rss?q=%23susi into your rss reader. Oh
+http://loklak.org/api/search.rss?q=%23susi into your RSS reader. Oh
```
Comment (aashutoshrathi) [inline README.rst]: Remove the space before `Oh` too.

File: README.rst [hunk 2]
```diff
@@ -186,7 +186,7 @@ http://loklak.org/api/search.rss?q=0rb1t3r or just follow @0rb1t3r
 Where can I find documentation?
 -------------------------------
 
-The Doumentation can be found at https://github.com/fossasia/susi_server/tree/development/docs/api/api.md
+The Documentation can be found at https://github.com/fossasia/susi_server/tree/development/docs/api/api.md
```
Comment (aashutoshrathi) [inline README.rst]: Make it can be found [here](https://github.com/fossasia/susi_server/tree/development/docs/api/api.md)

File: README.rst [hunk 3]
```diff
@@ -198,7 +198,7 @@ Where can I report bugs and make feature requests?
 --------------------------------------------------
 
 This project is considered a community work. The development crew
-consist of you too. I am very thankful for pull request. So if you
+consists of you too. I am very thankful for pull request. So if you
```
Comment (aashutoshrathi) [inline README.rst]: Make it `we are very thankful`, you know as its message from Org.

File: README.rst (hunk 1 above)
Comment (girijamanojkumarreddy) [reply README.rst]: I think space is required over there because of the full stop.

File: README.rst (hunk 2 above)
Comment (girijamanojkumarreddy) [reply README.rst]: I have done that now and thanks for bringing that into notice

File: README.rst (hunk 3 above)
Comment (girijamanojkumarreddy) [reply README.rst]: As its message from Org, I have changed the entire paragraph into first person plural.

**Pull request merged**